File: src/main/java/com/sudosoo/takeiteasy/service/CouponService.java
package com.sudosoo.takeiteasy.service;

import com.sudosoo.takeiteasy.dto.event.CreateEventRequestDto;
import com.sudosoo.takeiteasy.entity.Coupon;

public interface CouponService {

    Coupon createCoupon(CreateEventRequestDto requestDto);

}

File: takeItEasy-application/src/main/kotlin/com/sudosoo/takeItEasy/service/CouponService.kt
package com.sudosoo.takeItEasy.service

import com.sudosoo.takeItEasy.dto.event.CreateEventRequestDto
import com.sudosoo.takeiteasy.entity.Coupon

interface CouponService {
    fun priceCouponCreate(requestDto: CreateEventRequestDto): Coupon
    fun rateCouponCreate(requestDto: CreateEventRequestDto): Coupon
}
